ripgrep/.gitpod.yml

```diff
@@ -0,0 +1,8 @@
+image:
+  file: .gitpod.Dockerfile
+tasks:
+- init: cargo build
+  command: cargo test --all
+- command: cargo watch -x check
+  openIn: bottom
```

Comment:
Isn't that the default value?

Reply:
I think we'll need another iteration, will fix then.